Roadmap.md

```diff
@@ -1,188 +1,235 @@
 
-1. Automatic selection of development vs. production build of React (completed)
-2. Abstraction for isomorphic server-side rendering ([ongoing](https://github.com/meteor/meteor/blob/devel/packages/server-render/README.md))
-3. Integration of [dynamic imports](https://blog.meteor.com/dynamic-imports-in-meteor-1-5-c6130419c3cd) with React SSR
-4. Full support for optimized CSS-in-JS features of libraries like [styled-components](https://www.styled-components.com/)
+As with any roadmap, this is a living document that will evolve as priorities and dependencies shift; we aim to update the roadmap with any changes or status updates every quarter.
 
-We think Meteor has a clear set of benefits when compared to other popular React frameworks like Create React App and Next.js.
+Contributors are encouraged to focus their efforts on work that aligns with the roadmap then we can work together in these areas.
 
-## Remove blockers to Meteor adoption
+PRs to the roadmap are welcome. If you are willing to contribute please open a PR explaining your ideas and what you would be able to do yourself.
 
-### Make the `meteor` command-line tool installable from npm
+Ideally, every item in this roadmap should have at least two leaders, leaders are people that are interested in the item and would like to help. If you are interested please open a PR including yourself and describing how do you want to help.
 
-Installing `meteor` from npm would enable developers to use it as build tool for npm-based projects, and would simplify the Meteor release process by getting rid of the "dev bundle" (essentially the npm dependencies of the command-line tool).
+## Core
+### Update to Node.js 12
+- Leaders: [Ben Newman](https://github.com/benjamn)
+- Status: In Progress
+- PRs: https://github.com/meteor/meteor/pull/10527
 
-The biggest blockers to this project are
+Since Node.js 12 is scheduled to become the LTS version on October 1st, 2019, Meteor 1.9 will update the Node.js version used by Meteor from 8.16.1 (in Meteor 1.8.2) to 12.10.0 (the most recent current version).
 
-1. deciding whether/how to preserve Meteor release versions, and
-2. changing the API of the package server so that you don't have to download the entire package database locally.
+### Tree Shaking
+- Leaders: [Ben Newman](https://github.com/benjamn)
+- Status: -
+- PRs: -
 
-## Page load performance improvements
+Implement tree shaking / dead code elimination, which involves pruning the dependency tree while scanning imports in the `ImportScanner`. We believe it should be possible to treat values like `Meteor.isProduction` as constants during this process, and eliminate those branches if their conditions are false, so that the `./routes` dependency would not show up at all in the production bundle.
```

Comment:
Could also be done for `isServer`/`isClient`. But you can only eliminate these branches if everything within them is block-scoped. Function-scoped variable declarations are hoisted outside of them, so you may break stuff if you remove the whole branch.

Reply:
That's the kind of thing that a good minifier can figure out, once we tell it to consider expressions like `Meteor.isServer` as constants—which I agree is a great idea!